src/main/java/seedu/programmer/commons/util/JsonUtil.java

```diff
@@ -186,15 +187,15 @@ public static void writeJsonToCsv(JSONArray jsonData, File destinationFile) {
      * @param filePath Path of file to get JSON data from.
      * @return JSONArray of student's data.
```

Comment:
I think can all the Exception thrown in the javadoc too

Reply:
Good catch!